deps/libgit2.mk

```diff
@@ -47,6 +47,10 @@ OPENSSL_DIR=$(shell openssl version -d | cut -d '"' -f 2)
 ifeq ($(shell [ -e $(OPENSSL_DIR)/cert.pem ] && echo exists),exists)
 CERTFILE=$(OPENSSL_DIR)/cert.pem
 endif
+# This certfile location observed on Ubuntu 16.04
+ifeq ($(shell [ -e $(OPENSSL_DIR)/certs/ca-certificates.crt ] && echo exists),exists)
```

Comment:
Sorry, this was needed for Julia to build on my machine. Now that the PR has been merged, I will rebase.